docs/reference/tab-widgets/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -0,0 +1,211 @@
+// tag::cloud[]
+In order to restore the indices and data streams that are missing data:
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment.
++
+
+NOTE: If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support].
+If your deployment doesn't include {kib}, all you need to do is
+{cloud}/ec-access-kibana.html[enable it first].
++
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Stack Management > Index Management**.
++
+. In this step we are going to find all the indices and data streams missing data. If you already have that information
+you could skip this. Go in the **Indices** tab, enable **Include hidden indices** toggle and find all the open indices
+that have an empty **Health** and empty **Data stream**, keep their names somewhere handy. Then, go to the
+**Data Streams** tab and keep the name of all the data streams whose **Health** is `red`.
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-index-management-select-red-indices.png[{kib} indices,align="center"]
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-index-management-select-red-data-steams.png[{kib} data streams,align="center"]
++
+. Use the left side menu to go to **Snapshot and Restore**, there you can verify that you have a snapshot to restore
+the target indices and data streams from. By clicking on every snapshot you can see the indices and data streams it
+contains. Not all resources need to be restored from the same snapshot. **Do not proceed** there is no workable
```

Comment:
```suggestion
contains. Not all resources need to be restored from the same snapshot. **Do not proceed** if there is no workable
```

Reply:
I find this a bit confusing - what does "workable backup" mean here? An up-to-date enough backup or the index existing at all in the snapshot? 

Should we remove it or be more precise?

Reply:
I think we should make it more precise, it is good to warn the user about permanent implications.